cpp/models/abm/parameters.h

```diff
@@ -224,10 +224,43 @@ struct SocialEventRate {
     }
 };
 
+struct GotoWorkTimeMinimum {
+    using Type = CustomIndexArray<int, AbmAgeGroup>;
```

Comment:
I think this could be `TimeSpan` instead of `int`. Semantically more consistent.

Reply:
This is done.